ludwig/utils/strings_utils.py

```diff
@@ -258,7 +258,8 @@ def create_vocabulary(
             vocab = [padding_symbol] + vocab
         else:
             padding_symbol = pad_token
-
+    elif tokenizer_type in {"clip_tokenizer", "gpt2bpe_tokenizer"}:
```

Comment:
Perhaps we can make this a little more generic:

```
elif hasattr(tokenizer, "get_vocab"):
    vocab = tokenizer.get_vocab()
```

Reply:
Done.